@AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_sign.sqf

```diff
@@ -1,13 +1,18 @@
 /* ----------------------------------------------------------------------------
 Function: fza_fnc_sign
+
```

Comment:
Are these changes required?

Reply:
Keeps the documentation formatting inline with the rest of the documentation formatting.